test/xprof_http_e2e_SUITE.erl

```diff
@@ -3,7 +3,7 @@
 -include_lib("common_test/include/ct.hrl").
 -include_lib("eunit/include/eunit.hrl").
 
--compile(export_all).
+-compile([export_all, nowarn_export_all]).
```

Comment:
TBH: I don't like that approach in _CT_, can't we export everything explicitly?

Reply:
ok, I was hesitating too. just didn't want to duplicate long testcase name list. (basically apart from some helper functions all functions need to be exported)